doc/onboarding.md

```diff
@@ -0,0 +1,12 @@
+## Onboarding
+
+The following instructions are for 18F developers working on this project. This
+document should contain all steps required to get up and running on
+Micropurchase.
+
+### Add yourself as a GitHub Admin
```

Comment:
Admin access allows you to:
- create/edit auctions
- read users
- view bids when auctions are running (this info is veiled/sealed when the auctions are running)